src/mono/wasm/runtime/binding_support.js

```diff
@@ -497,7 +497,7 @@ var BindingSupportLib = {
 			// you need to use a view. A view provides a context — that is, a data type, starting offset, 
 			// and number of elements — that turns the data into an actual typed array.
 			// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Typed_arrays
-			if (!!(js_obj.buffer instanceof ArrayBuffer && js_obj.BYTES_PER_ELEMENT)) 
+			if (!!((js_obj.buffer instanceof ArrayBuffer || js_obj.buffer instanceof SharedArrayBuffer) && js_obj.BYTES_PER_ELEMENT))
```

Comment:
```suggestion
			if (!!((js_obj.buffer instanceof ArrayBuffer ||  (typeof SharedArrayBuffer !== "undefined" && js_obj.buffer instanceof SharedArrayBuffer)) && js_obj.BYTES_PER_ELEMENT))
```
This should fix the exception from the `instanceof` I believe

Reply:
Thanks @benaadams for the suggested change.  I did it just a little different but please let me know if this works or you have a problem with the way I approached it.